src/utils/src/structs.rs

```diff
@@ -41,13 +44,16 @@ pub unsafe fn read_struct<T: Copy, F: Read>(f: &mut F, out: &mut T) -> Result<()
 
 /// Reads an array of structs from an input buffer.  Returns a Vec of structs initialized with data
 /// from the specified input.
-/// This is unsafe because the structs are initialized to unverified data read from the input.
-/// `read_struct_slice` should only be called for plain old data structs.  It is not endian safe.
 ///
 /// # Arguments
 ///
 /// * `f` - The input to read from.  Often this is a file.
 /// * `len` - The number of structs to fill with data read from `f`.
+///
+/// # Safety
+///
+/// This is unsafe because the structs are initialized to unverified data read from the input.
+/// `read_struct_slice` should only be called for plain old data structs.  It is not endian safe.
 // This lint check is now deprecated - https://github.com/rust-lang/rust-clippy/pull/3478/files
```

Comment:
Is this comment still valid? What lint check is it refferring to?

Reply:
Removed. Looks like that comment had been left behind when the clippy override was removed.